client/app/components/dashboards/AddWidgetDialog.jsx

```diff
@@ -9,6 +9,7 @@ import {
   ParameterMappingListInput,
   editableMappingsToParameterMappings,
 } from '@/components/ParameterMappingInput';
+import ModelTagsControl from '@/components/tags-control/ModelTagsControl';
```

Comment:
Why this one instead of [QueryTagsControl](https://github.com/getredash/redash/blob/master/client/app/components/tags-control/QueryTagsControl.jsx)?